src/content/manifest/overview-manifest.mdx

```diff
@@ -0,0 +1,153 @@
+
+## Destroy
+
+The commands you define in this section will ensure any resources external to the Kubernetes cluster Okteto manages are destroyed when deleting the development environment. 
+
+> Note: The Okteto CLI command `okteto destroy` will automatically destroy any Kubernetes resources created by the `okteto deploy` command.
+
+Your `destroy` section might look like this:
+
+```yaml
+destroy:
+  image: okteto/pipeline-runner:1.0.0-sam
+  commands:
+    - name: destroy worker service
+      command: |
+        sam delete --no-prompts --stack-name "${OKTETO_NAMESPACE}-voting-texkhnclxd" --region us-east-1
+```
+
+In this example `destroy` section we are deleting the worker service running on the image container specified in the `image:` attribute. When the development environment is deleted, Okteto will execute this command ensuring orphaned resources don't remain active.
+
+## External
+
+This section contains a list of resources external to the Okteto cluster such as cloud resources or dashboards, but that are part of your development environment. Configuring your external resources enables you to build a more complete development experience in Okteto. 
+
+Your `external` section might look like this:
+
+```yaml
+external:
+  readme:
+    icon: okteto
+    notes: README.md
+    endpoints:
+    - name: Try it out!
+      url: https://github.com/okteto/voting-app-with-external-resources
+  lambda:
+    icon: aws
+    notes: docs/lambda.md
+    endpoints:
+    - name: function
+```
+
+This section provides only the metadata to represent and link to external resources in the Okteto UI and does not create any resources in a cloud provider. Creating or destroying the resources themselves is done in the [deploy](#deploy) and [destroy](#destroy) sections, respectively.
+
+## Dev
+
+This section contains a list of instructions that determine how the Okteto CLI and, specifically, the `okteto up` command behave. This section is how Okteto will synchronize the code between your local computer and the remote development container in Okteto.
+
+Your `dev` section might look like this:
+
+```yaml
+dev:
+  api:
+    command: ["bash"]
+    forward:
+      - 8080:8080
+      - 9229:9229
+    sync:
+      - api:/usr/src/app
+  frontend:
+    command: yarn start
+    sync:
+      - frontend:/usr/src/app
+```
+
+In this example, the `api` service you define is *separate* from the original `api` service created in the `build` and `deploy` sections. This `api` service in the `dev` section will tell Okteto to create a new development container that exists specifically to sync code between your local computer and the remote Okteto development container so you can see your changes in real-time within a production-like environment. This also enables you to share an endpoint for others to view and collaborate on your work with you without having to do any extra work.
```

Comment:
Should we add a link to the `dev` reference docs?

Reply:
```suggestion
In this example, the `api` service you define is *separate* from the original `api` service created in the `build` and `deploy` sections. This `api` service in the `dev` section will tell Okteto to create a new development container that exists specifically to sync code between your local computer and the remote Okteto development container so you can see your changes in real-time within a production-like environment. This also enables you to share an endpoint for others to view and collaborate on your work with you without having to do any extra work.

You can learn more about this section [here](reference/okteto-manifest.mdx#dev-object-optional).
```